src/libs/NextStepUtils.ts

```diff
@@ -125,6 +135,12 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
                 ],
             };
 
+            if (!PolicyUtils.arePaymentsEnabled(policy)) {
```

Comment:
Sorry one more thing, @koko57 do you think you could explain why you added this? If payments are disabled and there is some non instant auto reporting frequency (weekly, monthly, etc.), the next steps should still be "Waiting for x's expenses to automatically submit" (below text). cc @trjExpensify for confirmation.

Reply:
A delayed submission frequency other than `manually`, yes.

Reply:
@srikarparsi I understood that it also must be that way. So I will remove it